bin/run_demo.sh

```diff
@@ -54,3 +54,12 @@ pipetask run -d "exposure=903342 AND detector=10" -j 1 -b DATA_REPO/butler.yaml
     -i HSC/calib,HSC/raw/all,ref_cats,shared/ci_hsc \
     --register-dataset-types -p "${PIPE_TASKS_DIR}/pipelines/ProcessCcd.yaml" \
     --instrument lsst.obs.subaru.HyperSuprimeCam --output-run demo_collection
+
+# Do not specify a number of processors (-j) to test that the default value
+# works.
+# Do not provide a data query (-d) to verify code correctly handles an empty
+# query.
+pipetask run -b DATA_REPO/butler.yaml \
```

Comment:
Am I right in thinking that this could be `qgraph` rather than `run` and still test the empty query code path? If so then it would be better to do that and not spend the time actually processing the data again.

Reply:
`-j` is only for `run`. But, I could remove the `-j` on line 53, especially since it's only indicating the default value (1), and change this to `qgraph`.